tools/install_fedora_deps.sh

```diff
@@ -72,12 +72,6 @@ cd behavioral-model
 ./configure --with-pdfixed --with-thrift --with-pi --with-stress-tests --enable-debugger
 make
 make install-strip
-
-cd targets/simple_switch_grpc/
```

Comment:
this stuff is not needed?

Reply:
This step is no longer necessary because the `configure.ac` file for simple_switch_grpc has been integrated with the `configure.ac` file in the top-level directory.